server/routes/payment.js

```diff
@@ -129,9 +137,26 @@ router.get('/success', (req, res) => {
                 }
                 else {
                     let activity = new Activity({ date: new Date(), amount: amount.total, name: "Associated_account" })
-                    activity.save().catch((err) => {
-                        console.log(err)
-                        throw err
+                    /*  activity.save().catch((err) => {
+                         console.log(err)
+                         throw err
+                     }) */
+                    /**
+                     * Gets a student by ID and adds the activity to the student's activity list
+                     */
+                    Student.findOne({ id: studentId }, (err, results) => {
+                        if (err) {
+                            console.log(err.response);
```

Comment:
When we're logging, we want to log it to /var/log/ on Heroku. I'm not sure if this is how it should be done, but we can discuss this later today.

Reply:
Yeah, I'll be trying to work on that today. Right now I just pushed the payment integration for back-end.